libs/src/index.js

```diff
@@ -107,7 +107,7 @@ class AudiusLibs {
    * @param {string} registryAddress
    * @param {string | Web3 | Array<string>} providers web3 provider endpoint(s)
    */
-  static configInternalWeb3 (registryAddress, providers) {
+  static configInternalWeb3 (registryAddress, providers, privateKey) {
```

Comment:
@hareeshnagaraj not sure if this solves the same problem, but @cheran-senthil passed in a wallet that could sign eth transactions like this https://github.com/AudiusProject/audius-k8s/pull/177/files#diff-6d777c865f2f6389dcaaf388a6fc08eef5466dddebd4058944271783450a2663R37-R45

Reply:
@SidSethi to confirm my understanding here - i think this actually does solve the same problem, however we want to be 'logged in' to the dataContracts web3 object for the reconfiguration operations later so unless we want to do this for every single operation requiring a signature we should keep the current pattern

Reply:
doing it this way also incurs a call to the provider with `ethSignTypedData` instead of just using sig-util

Reply:
just so i'm clear, the purpose of this pkey is for to get the correct ownerWallet here (https://github.com/AudiusProject/audius-protocol/pull/1141/files#diff-0a02a35fbb734317785e1cd8180f6042c116e0d5e13482d6b5ac4bce16d02916R53-R58) so the ethSignTypedData has the right data here(https://github.com/AudiusProject/audius-protocol/pull/1141/files#diff-0a02a35fbb734317785e1cd8180f6042c116e0d5e13482d6b5ac4bce16d02916R135-R139)?

Reply:
yes, also required for reconfiguration submission in the upcoming work - right now is only used in addOrUpdateContentNode signed data generation. actually allows a private key to be used as the wallet in dataContracts, something this solution from cheran would not allow - the benefit here is that we don't have to make a remote web3 provider call to actually use `ethSignTypedData` (useExternalWeb3 becomes false in content node), and can just use sig-util. this is already done in web3Manager